pkg/detectors/langfuse/langfuse.go

```diff
@@ -0,0 +1,111 @@
+package langfuse
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+	"encoding/base64"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	publicKey = regexp.MustCompile(`\b(pk-lf-[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`)
+	secretKey = regexp.MustCompile(`\b(sk-lf-[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`)
+)
+
+func (s Scanner) Keywords() []string {
+	return []string{"langfuse", "pk-lf-", "sk-lf-"}
```

Comment:
In my opinion, `pk-lf-` and `sk-lf-` are sufficient as keywords. They meet our criteria of having at least four unique characters.